crates/eth-rpc/src/servers/net_rpc.rs

```diff
@@ -23,12 +22,10 @@ impl NetApiServer for NetRpc {
     }
 
     fn peer_count(&self) -> Result<PeerCount> {
-        // TODO: replace jsonrpsee::types::ErrorObject with Provider::Error once NetRpc uses Provider
-        Err(EthApiError::<jsonrpsee::types::ErrorObject>::MethodNotSupported("eth_peerCount".to_string()).into())
+        Ok(PeerCount::Number(0))
```

Comment:
Rather than having a hardcoded constant, better to have a lazy_static for this, like we have [here](https://github.com/kkrt-labs/kakarot-rpc/blob/50fc732165871d13b1acb2eb81778b75b14c0beb/crates/core/src/client/constants.rs#L93C5-L93C5).

So it can be:

```rust
lazy_static! {
  pub static ref PEER_COUNT: PeerCount  = PeerCount::Number(0) 
 }
```

Reply:
and then you can use this lazy_static in place of the hardcoded value

Reply:
ahh yes, good suggestion, done!